/**:
  ros__parameters:
    base_frame_id: os1_sensor 
    map_frame_id: map

    scan_voxel_size: 2.0
    map_voxel_size: 0.2
    crop_min_x: -1.0
    crop_max_x: 1.0
    crop_min_y: -1.0
    crop_max_y: 1.0
    crop_min_z: -1.0
    crop_max_z: 1.0

    transformation_epsilon: 0.01
    step_size: 0.1
    resolution: 3.0
    max_iteration: 30
    omp_num_thread: 4

